pythnet-price-feeds/cosmwasm.md

```diff
@@ -0,0 +1,28 @@
+---
+description: Consume Pyth Network prices in applications on Cosmwasm
+---
+
+# Pyth on Cosmwasm
+
+Cosmwasm contracts can update and fetch the Pyth prices using the Pyth Cosmwasm Contract, which has been deployed on Testnet. The documented source code can be found [here](https://github.com/pyth-network/pyth-crosschain/tree/main/target-chains/cosmwasm/contracts/pyth).
```

Comment:
suggest replacing "which has been deployed on Testnet" with "deployed on their network". This contract will live on many different mainnets and testnets in the future, so may as well write the text to plan for that future.